### PR TITLE
Loading spinner moved to left alignment

### PR DESCRIPTION
**Pull request by matildepark**
Upon @galenwp's request.

Screen Shot 2019-08-09 at 6 43 34 PM

This PR distinctly does *not* include the compiled pkg/arvo code, in the hopes of establishing a brighter future. It can easily be added as a separate commit if need be.

### CODE REVIEW

File: pkg/interface/chat/src/js/components/lib/header-bar.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export class HeaderBar extends Component {
   render() {
     let spin = (this.props.spinner)
       ?  <div className="absolute"
-           style={{width: 16, height: 16, top: 16, right: 16}}>
+           style={{width: 16, height: 16, top: 16, left: 55}}>
```
Comment (matildepark) [inline pkg/interface/chat/src/js/components/lib/header-bar.js]: I feel weird using an odd number given the pattern here. Is 64px what we really want?

Comment (matildepark) [reply pkg/interface/chat/src/js/components/lib/header-bar.js]: That would look like this btw

image

**Comment by galenwp** [comment]
Thank you! I'm totally unfamiliar with the grid in Modulo — so can't advise about the spacing. I'd say: just have the spacing between the home button and the spinner be the same as between the home button and the left edge?

**Comment by matildepark** [comment]
That would be about 8px spacing, which would look like this. I'll wait on @g-a-v-i-n's confirmation here.

Screen Shot 2019-08-09 at 7 39 00 PM

**Pull request merged**